snapshot/example/ExampleMacOS/AppDelegate.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AppDelegate.swift
+//  ExampleMacOS
+//
+//  Created by Alexander Semenov on 1/19/17.
+//  Copyright © 2017 Felix Krause. All rights reserved.
```

Comment:
We should figure out what to do about these copyright statements. I don't think they should stay.